lib/cc/engine/analyzers/violation.rb

```diff
@@ -32,12 +39,24 @@ def report_name
           "#{current_sexp.file}-#{current_sexp.line}"
         end
 
+        def mass # ABH Reverse engineer the actual mass from flay score.
```

Comment:
Can we just use the `#mass` method on the `Sexp`? This code looks like it's trying to reverse engineer flay's internal score calculation from `Sexp#mass`, but we have the `Sexp` available here.